### ISSUE TITLE
Add 'edit' and 'destroy' on lm details page

### ISSUE THREAD

**Issue by emcoding** [TeamCheesy/assessment#7]
@roosdebildt For you?!

**Comment by roosdebildt** [comment]
if, at this point, it is still ok if those options are not  secured/open I can do it!!

**Comment by emcoding** [comment]
Not sure if I understand. We are talking about implementing edit and destroy, Right?

**Comment by roosdebildt** [comment]
yes, so if it's just about creating those actions, I can do it. But normally you want to make sure not everybody can edit and destroy. but if at this point, that is no problem, I can do it :)
